lib/routes/account.js

```diff
@@ -544,19 +541,26 @@ module.exports = function (
         }
 
         function sendVerifyAccountEmail() {
-          // For legacy clients, behavior is to not send an email to verify their account
-          // and have the requestor send it.
-          var sendEmailIfUnverified = false
-
-          // If a value was passed, use that instead
-          if (request.payload.sendEmailIfUnverified !== undefined) {
-            sendEmailIfUnverified = request.payload.sendEmailIfUnverified
+          // Delegate sending emails for unverified users to auth-server.
+          // Will be removed once all clients have been updated not to send verify emails.
+          // https://github.com/mozilla/fxa-auth-server/issues/1325
+          sendEmailIfUnverified = request.query.sendEmailIfUnverified
+          emailSent = false
+
+          // For all non content-server requests, default to sending verify email on login
```

Comment:
The implicit info that should probably be stated explicitly is that only the content server sends a `metricsContext`